xsnap/sources/xsnap.c

```diff
@@ -294,9 +324,8 @@ void xsBuildAgent(xsMachine* machine)
 	xsResult = xsNewHostFunction(xs_issueCommand, 1);
 	xsDefine(xsGlobal, xsID("issueCommand"), xsResult, xsDontEnum);
 	
-	xsVar(0) = xsGet(xsGlobal, xsID("Date"));
-	xsVar(0) = xsGet(xsVar(0), xsID("now"));
```

Comment:
this looks like a git-oopsie.

@kriskowal to take a look.

Reply:
I haven’t traced the diff confusion, but the end state is that both xsnap.c and xsnap-worker.c use the same high resolution timer native and that this `performance.now = Date.now` stop-gap has been converted.